libs/components/code-examples/src/lib/modules/action-bars/summary-action-bar/error/example.component.ts

```diff
@@ -0,0 +1,81 @@
+import { Component, inject } from '@angular/core';
+import {
+  FormBuilder,
+  FormControl,
+  FormGroup,
+  ReactiveFormsModule,
+} from '@angular/forms';
+import {
+  SkySummaryActionBarError,
+  SkySummaryActionBarModule,
+} from '@skyux/action-bars';
+import { SkyKeyInfoModule } from '@skyux/indicators';
+
+interface donationSummary {
+  value: number;
+  label: string;
+}
+
+/**
+ * @title Summary action bar with errors
+ * @docsDemoHidden
+ */
+@Component({
+  selector: 'app-action-bars-summary-action-bar-error-example',
+  templateUrl: './example.component.html',
+  imports: [SkyKeyInfoModule, SkySummaryActionBarModule, ReactiveFormsModule],
+})
+export class ActionBarsSummaryActionBarErrorExampleComponent {
+  public formErrors: SkySummaryActionBarError[] = [];
+  public exampleForm: FormGroup;
+  public donationSummary: FormControl<donationSummary[] | null>;
+
+  constructor() {
+    this.donationSummary = new FormControl<donationSummary[] | null>([
+      {
+        value: 250,
+        label: 'Given this month',
+      },
+      {
+        value: 1000,
+        label: 'Given this year',
+      },
+      {
+        value: 1300,
+        label: 'Given all time',
+      },
+    ]);
+    this.exampleForm = inject(FormBuilder).group({
+      donationSummary: this.donationSummary,
+    });
+
+    this.donationSummary.statusChanges.subscribe(() => {
```

Comment:
Needs a `takeUntilDestroyed()`? Is there potential for this subscription to be a signal so that you can call `formErrors()` in the template?

Reply:
I sadly do not this is possible bc converting `statusChanges` or `events` to a signal only updates when the value changes or when the status goes from `VALID` to `INVALID` so if there is one error, and then two more errors are added the signal doesnt update 😞 I have no idea why there isnt an observable to hook into for errors changing